core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -59,15 +59,15 @@ enum class UniqueType(val text:String, vararg targets: UniqueTarget) {
 
     //////////////////////////////////////// GLOBAL UNIQUES ////////////////////////////////////////
 
-    
+
     /////// Stat providing uniques
 
-    Stats("[stats]", UniqueTarget.Global),
+    Stats("[stats]", UniqueTarget.Global, UniqueTarget.FollowerBelief),
```

Comment:
In my opinion this should also be `UniqueTarget.Building`

Reply:
UniqueTarget.Building is included in Global, but FollowerBelief was/is not - FollowerBelief(Global) would have been the other solution AFAIK, but this seemed more - defensive?